src/api/reactivity-utilities.md

```diff
@@ -102,7 +102,7 @@ Can be used to create a ref for a property on a source reactive object. The crea
 
 ## toRefs() {#torefs}
 
-Converts a reactive object to a plain object where each property of the resulting object is a ref pointing to the corresponding property of the original object. Each individual ref is created using [`toRef()`](#toref).
+Converts a object to a plain object where each property of the resulting object is a ref pointing to the corresponding property of the original object. Each individual ref is created using [`toRef()`](#toref).
```

Comment:
I believe the object passed to `toRefs` does need to be reactive. There's a warning logged if it isn't:

<https://github.com/vuejs/core/blob/f3e4f038bf299794b2ca4750b4678074660113c8/packages/reactivity/src/ref.ts#L205>

Reply:
Thanks for your reply, I just read the ref.ts and got it.
Now I wonder if "a reactive source object" is exactly correct in the description of "toRef".

Reply:
> I believe the object passed to `toRefs` does need to be reactive. There's a warning logged if it isn't:
> 
> https://github.com/vuejs/core/blob/f3e4f038bf299794b2ca4750b4678074660113c8/packages/reactivity/src/ref.ts#L205

Thanks for your reply, I just read the ref.ts and got it.
Now I wonder if "a reactive source object" is exactly correct in the description of "toRef".